auth/providers/azure/graph/graph_test.go

```diff
@@ -107,11 +107,12 @@ func TestLogin(t *testing.T) {
 		validBody := `{
   "token_type": "Bearer",
   "expires_in": 3599,
-  "access_token": "%s"
+  "access_token": "%s",
```

Comment:
you should then update this paylod to remove the `expired_in` and anywhere else

Reply:
Updated